Gemfile

```diff
@@ -9,6 +9,7 @@ gem 'acts_as_tree'
 gem 'cancancan', '~> 1.10'
 gem 'devise'
 gem 'figaro'
+gem 'thumbs_up', github: 'bouchard/thumbs_up', ref: 'f499a7'
```

Comment:
Any reason we're not using the straight-up `gem 'thumbs_up'` as suggested on the [gem homepage](https://github.com/bouchard/thumbs_up)?